xmtp_api_d14n/src/protocol/traits/visitor.rs

```diff
@@ -103,6 +116,11 @@ pub trait EnvelopeVisitor<'env> {
         Ok(())
     }
 
+    fn visit_v3_welcome_pointer(&mut self, _m: &V3WelcomePointer) -> Result<(), Self::Error> {
+        tracing::trace!("noop_visit_v3_welcome_pointer");
+        panic!()
```

Comment:
i'm guessing this was for testing -- should remove/error

Reply:
Yeah, this was for testing, however, I'm kind of wondering if this should be the default so that we know that we're never calling methods that aren't implemented.